next/components/atoms/OrgStructurePersonCard.tsx

```diff
@@ -1,7 +1,8 @@
 import { AccordionCard } from '@bratislava/ui-bratislava/AccordionCard/AccordionCard'
-import { userDetailsFetcher } from '@utils/organisationalStructure'
 import useSWR from 'swr'
 
+import { userDetailsFetcher } from '../../backend/utils/organisationalStructure'
```

Comment:
```suggestion
import { userDetailsFetcher } from 'backend/utils/organisationalStructure'
```
+ after suggested change please sort imports